src/main/java/exception/InvalidFileDataException.java

```diff
@@ -0,0 +1,19 @@
+package exception;
+
+import java.io.File;
+
+public class InvalidFileDataException extends Exception {
+
+    private File cheatSheetDocument;
+
+    public InvalidFileDataException(File cheatSheetDocument) {
+        super();
```

Comment:
Not really sure about the use of super here since DirectoryIsEmptyException didn't do it